src/nwb_benchmarks/scripts/generate_figures.py

```diff
@@ -0,0 +1,335 @@
+import re
+import textwrap
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import polars as pl
+import seaborn as sns
+
+from nwb_benchmarks.database._processing import repackage_as_parquet
+
+# use lbl-mac machine_id for initial comparisons
+lbl_mac_id = "87fee773e425b4b1d3978fbf762d57effb0e8df8"  # "SincereCornucopia"
+
+# create new database file with latest results in local github repository
+results_directory = Path.home() / ".cache" / "nwb-benchmarks" / "nwb-benchmarks-results"
+db_directory = Path.home() / ".cache" / "nwb-benchmarks" / "nwb-benchmarks-database"
+output_directory = Path(__file__).parent / "figures"
+
+repackage_as_parquet(
+    directory=results_directory,
+    output_directory=db_directory,
+    minimum_results_version="3.0.0",
+    minimum_machines_version="1.4.0",
+)
```

Comment:
I think this is good for development. When paper is finalized we can remove the endpoint from the Flask server (or I guess do that whenever) and treat the 'results database' repo as a version controlled 'data' to go along with publication (also with Zenodo DOI)